Ujaan/Lab-3/flake.nix

```diff
@@ -0,0 +1,19 @@
+{
+  description = "Protobuf UDP Communication";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs";
+  };
+
+  outputs = { self, nixpkgs }:
+    let
+      system = "x86_64-linux";
+      pkgs = import nixpkgs { inherit system; };
+    in
+    {
+      defaultPackage.${system} = pkgs.mkShell {
+        nativeBuildInputs = [ pkgs.cmake ];
+        buildInputs = [ pkgs.protobuf pkgs.gcc ];
+      };
+    };
+}
```

Comment:
you will need a differently setup flake for lab 3, see the wiki for details:

https://wiki.hytechracing.org/books/software/page/lab-3-getting-started-with-protobuf-ssh-and-software-deployment#bkmrk-on-the-linux-command